.github/actions/pr-labels/src/index.js

```diff
@@ -74,102 +71,12 @@ async function run() {
     ).data.map((label) => label.name);
     core.debug(`Found ${labels.length} labels: ${labels.join(", ")}`);
 
-    // Get PR details to check for semantic commit messages
-    const prNumber = github.context.issue.number;
-    core.debug(`Processing PR #${prNumber}`);
-
-    core.debug("Fetching PR details...");
-    const { data: pullRequest } = await octokit.rest.pulls.get({
-      ...github.context.repo,
-      pull_number: prNumber,
-    });
-
-    // Get the PR title and HEAD commit message
-    const prTitle = pullRequest.title;
-    core.debug(`PR title: "${prTitle}"`);
-
-    // Get the HEAD commit message
-    core.debug("Fetching PR commits...");
-    const { data: commits } = await octokit.rest.pulls.listCommits({
-      ...github.context.repo,
-      pull_number: prNumber,
-    });
-
-    core.debug(`Found ${commits.length} commits in PR`);
-    const headCommitMessage = commits.length > 0 ? commits[commits.length - 1].commit.message : null;
-    if (headCommitMessage) {
-      core.debug(`HEAD commit message: "${headCommitMessage}"`);
-    } else {
-      core.debug("No HEAD commit message found");
-    }
-
-    // Try to extract semantic type from PR title or HEAD commit
-    core.debug("Extracting semantic type from PR title...");
-    const prTitleType = extractSemanticType(prTitle);
-
-    core.debug("Extracting semantic type from HEAD commit...");
-    const commitType = extractSemanticType(headCommitMessage);
-
-    // Use PR title type first, then fall back to commit type
-    const semanticType = prTitleType || commitType;
-    if (semanticType) {
-      core.debug(`Using semantic type: "${semanticType}"`);
+    // If labels already exist, skip adding new labels
+    if (labels.length > 0) {
+      core.info("Labels already exist on PR, skipping adding new labels");
```

Comment:
won't this include all labels, not just `type::` ones?

would it be possible to instead check if there's at least one `type::` label?

Reply:
Ya, sorry, I think I misunderstood the request.  I'll modify it.